devicetypes/smartthings/aeotec-wallmote.src/aeotec-wallmote.groovy

```diff
@@ -26,6 +26,7 @@ metadata {
         fingerprint mfr: "0086", model: "0082", deviceJoinName: "Aeotec Remote Control", mnmn: "SmartThings", vid: "generic-4-button" //Aeotec Wallmote Quad
         fingerprint mfr: "0086", model: "0081", deviceJoinName: "Aeotec Remote Control", mnmn: "SmartThings", vid: "generic-2-button" //Aeotec Wallmote
         fingerprint mfr: "0060", model: "0003", deviceJoinName: "Everspring Remote Control", mnmn: "SmartThings", vid: "generic-2-button" //Everspring Wall Switch
+		fingerprint mfr: "0371", model: "0016", deviceJoinName: "Aeotec Remote Control", mnmn: "SmartThings", vid: "generic-2-button" //Aeotec Wallmote 7
```

Comment:
- Could You use the same spacing as it's in this file (change 1 tab -> 4 spaces).
- Please add Raw Description (it can be found in the graphapi in device details)  in a comment at the end of this fingerprint -> // Aeotec Walmote 7, Raw description: [paste raw description from graphapi]